source/code/source/playfab/PlayFabAndroidHttpPlugin.cpp

```diff
@@ -562,6 +562,26 @@ namespace PlayFab
                 }
             }
 
+            std::string requestId = "initializingRequestId";
+            {
+                methodId = jniEnv->GetMethodID(GetHelper().GetHttpRequestClass(), "getRequestIdFromHeader","()[B");
+                if (methodId)
+                {
+                    jbyteArray requestIdBytes = (jbyteArray)jniEnv->CallObjectMethod(httpRequestObject, methodId);
+                    if(requestIdBytes != nullptr)
+                    {
+                        int requestIdSize = jniEnv->GetArrayLength(requestIdBytes);
+                        if(requestIdSize > 0)
+                        {
+                            std::vector<uint8_t> requestIdBuffer(static_cast<size_t>(requestIdSize));
+                            jniEnv->GetByteArrayRegion(requestIdBytes, 0, requestIdSize, reinterpret_cast<jbyte*>(requestIdBuffer.data()));
+                            std::string reqId(reinterpret_cast<const char*>(requestIdBuffer.data()), requestIdBuffer.size());
+                            requestId = reqId;
```

Comment:
Delete requestId temp variable, and change this line to:
requestContainer.SetRequestID(reqId)